### DISCUSSION TITLE
REI not showing recipes for any modded processes

### DISCUSSION THREAD

**Discussion by hiddenyak** [shedaniel/RoughlyEnoughItems#1175] [Q & A] [answered]
Only vanilla crafting recipes are shown, despite having many mods with their own recipes.

For example with thermal expansion:

![image](https://user-images.githubusercontent.com/50391075/198398699-f27bc88a-4108-4c7f-844e-b2fef659264a.png)

I am expecting to see multiple tabs at the top for other machines, etc

**Comment by shedaniel** [comment, marked as answer]
Did you install REI Plugin Compatibilities
